sdcm/tester.py

```diff
@@ -2961,16 +2961,28 @@ def clean_resources(self):
 
         self.destroy_credentials()
 
-    @silence(name='Save node schema')
     def save_nodes_schema(self):
-        if self.db_cluster is None:
-            self.log.info("No nodes found in the Scylla cluster")
+        def _save(node, silence_name, cmd, log_file):
+            with silence(name=silence_name, raise_error_event=False):
+                node.save_cqlsh_output_in_file(cmd=cmd, log_file=log_file)
+
+        cluster = self.db_cluster or self.k8s_cluster
```

Comment:
The `self.k8s_cluster` doesn't have `nodes` attr.
K8S cluster is different layer of abstraction.
The `self.db_cluster` attr is correct one for both cases.